content/information-aggregation/history-stats.js

```diff
@@ -107,7 +107,7 @@ Foxtrick.modules['HistoryStats'] = {
 				return;
 		}
 
-		var events = log.cloneNode(true).querySelectorAll('.otherEventText');
+		var events = log.cloneNode(true).querySelectorAll('td + td');
```

Comment:
this selector is bound to break
```suggestion
		var events = log.cloneNode(true).querySelectorAll('.feedItem .float_left');
```